src/controllers/notice.expired.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Session } from "ch-node-session-handler";
+import { NextFunction, Request, Response } from "express";
+import { Templates } from "../model/template.paths";
+import {
+  retrieveCompanyProfileFromObjectionSession,
+} from "../services/objection.session.service";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+  const session: Session = req.session as Session;
+
+  const companyName: string = retrieveCompanyProfileFromObjectionSession(session).companyName;
```

Comment:
you can shorten this line by using 
const { companyName } = retrieveCompanyProfileFromObjectionSession(session);
that says to extract companyName from retrieveCompanyProfileFromObjectionSession(session);
not sure if you'd need to add ": string" somewhere.